src/iidm/MergedBus.cpp

```diff
@@ -47,6 +47,21 @@ double MergedBus::getAngle() const {
     return stdcxx::nan();
 }
 
+stdcxx::CReference<Component> MergedBus::getConnectedComponent() const {
+    checkValidity();
+    for (const auto& it : m_buses) {
+        const auto& cc = it.get().getConnectedComponent();
```

Comment:
What is the return type of getConnectedComponent?

Reply:
it is a `stdcxx::Reference<Component>`. do you prefer using it explicitly ?